docs/docs/30-administration/22-backends/20-local.md

```diff
@@ -65,3 +65,11 @@ In the context of the local backend, plugins are simply executable binaries, whi
 > Default: default temp directory
 
 Directory to create folders for workflows.
+
+#### `WOODPECKER_BACKEND_LOCAL_EXEC_DIR`
```

Comment:
This PR seems pretty redundant to `WOODPECKER_LOCAL`.

Reply:
Well yes and no ...
... this is the most generic solution i did come up with to fix this.

The other var is not an engine option and only acept bool

Reply:
An other option would be to alter the backend config specific for this usecase ... which i found to be not as generic as this one